backend/routes/flightStatus.js

```diff
@@ -1,46 +1,247 @@
-import { API_KEY, API_SECRET } from "../config.js";
+import { API_KEY, API_SECRET, GOOGLE_MAPS_API_KEY } from "../config.js";
 import Amadeus from "amadeus";
 import express from "express";
-import { PrismaClient } from '@prisma/client'; 
+import { PrismaClient } from '@prisma/client';
 import { authenticateUser } from '../auth.js';
+import axios from 'axios';
 
 const prisma = new PrismaClient();
 const router = express.Router();
 const amadeus = new Amadeus({
-    clientId: API_KEY,
-    clientSecret: API_SECRET,
+clientId: API_KEY,
+clientSecret: API_SECRET,
 });
 
-//flight search for adding trip - needs authentication
+// Flight status endpoint
 router.get('/flight-status', authenticateUser, async (req, res) => {
-    try {
```

Comment:
Same here (nit), why did we get rid if tab spacing here? Is generally a good practice to keep code blocks indented.